scripts/setup.py

```diff
@@ -28,6 +40,28 @@ def setup(dataDir):
         filename = os.path.basename(file)
         path = os.path.join(dataDir, filename)
         open(path, "wb").write(response.content)
+    
+    # Loading the temp.zip and creating a zip object
+    with ZipFile("../data/vosk-model-small-en-us-0.15.zip", 'r') as zObject:
+    
+        # Extracting the zip
+        zObject.extractall(path="../data")
+    
+    # Delete the zip file
+    os.remove("../data/vosk-model-small-en-us-0.15.zip")
+
+    
+def install_packages():
+    """Install the required python libraries."""
+
+    for package in PACKAGES:
+        print("Downloading python library {}...".format(package))
+        try:
+            subprocess.check_call([sys.executable, "-m", "pip", "install", package], stdout=subprocess.DEVNULL)
+        except Exception as e:
+            print("Failed to download library {}".format(package))
+            print("Exception {}".format(e))
```

Comment:
Installing python packages is not so straight forward on my distro, I would prefer us adding a `requirements.txt` with the necessary packages and just install them with `pip`.